automode css

chromoxdor opened the pull request:
> We should start implementing it and if it may run into issues, we can maybe split the CSS into 2 parts so one part is excluded when we don't want to add the "dark mode" to save some space for example. Or we can make the CSS always be loaded from an external source for those builds that will fail as a result of this change. Another option can be to make the "dark mode" part always external, etc. Lots and lots of options.

Here the automode css as well as dark and light only files...

chromoxdor committed:
automode css

chromoxdor committed:
additional css

TD-er commented:
I applaud the idea to split it, but just merging the CSS into the main branch will not be very useful unless it can also be selected somewhere.

chromoxdor commented:
Thats right. But having the files already there doen´t hurt anyone.. 🙂

TD-er commented:
Have to check the changes you made in the default CSS, to see what has changed.
I can do a file diff, but that doesn't make it very clear (to me) what has changed, so have to try it myself. :)

chromoxdor commented:
I totally understand and appreciate your effort to check the changes but this css is heavily tested and in use for quite some time. 
So it is not my intention to make more work for you. I thought it is an integrate-and-forget type of situation...🙂

chromoxdor committed:
reverted some changes for now

i reverted this because the version before was changed unter the assumption that #3884 (cleaner css) was already integrated.
But since it is not, this version of my css is the one to go... (i will now commit changes when they are needed and not on predictions)

TD-er commented:
> I thought it is an integrate-and-forget type of situation...

It never is ;)
I always like to know what has been changed.
Just to give an idea of how I "program"... most of the programming is done while walking outside, in my head.
So when I return home, I must not be disturbed, run to my PC and start memory-dumping whatever I came up with during the walk. :)
But this requires me to continuously keep myself updated on what has changed.

The exception on this is JavaScript as I cannot program in this language without StackOverflow present and even then I just can't wrap my head around it.

chromoxdor commented:
> The exception on this is JavaScript as I cannot program in this language without StackOverflow present and even then I just can't wrap my head around it.


For me it is the same..but unfortunately it is the only programming language i "speak" 😁

TD-er commented:
I don't speak JavaScript, I only copy/paste it sentence by sentence.
As soon as StackOverflow is offline, I'm hopelessly lost in that language.
